Dockerfile

```diff
@@ -10,6 +10,7 @@ COPY --from=build-env /go/src/github.com/openshift/cluster-network-operator/_out
 COPY --from=build-env /go/src/github.com/openshift/cluster-network-operator/_output/linux/amd64/cluster-network-renderer /bin/cluster-network-renderer
 COPY manifests /manifests
 COPY bindata /bindata
+COPY testbindata /testbindata
```

Comment:
We really want test data in the image? Is that what other people are doing?

Reply:
I'm not sure. I haven't really decided how to do e2e testing. I could get rid of this you like.

Reply:
I don't really know...